app/views/secrets.html

```diff
@@ -14,28 +14,39 @@
           </span>
         </h1>
       </div>
+      <div ng-if="(secrets | hashSize) > 0 || filterWithZeroResults" class="data-toolbar">
```

Comment:
I think we could just check `unfilteredSecrets` size here. Note that we've started using `| size` instead of `| hashSize`.

Does this work?

```html
<div ng-if="unfilteredSecrets | size" class="data-toolbar">
```

Reply:
It does :+1: